### ISSUE TITLE
On opening webchat, it says : 'Looks like I’m all mixed up. Let’s try asking again, but maybe rephrase your request?' 

### ISSUE THREAD

**Issue by mareekuh** [microsoft/botframework-components#927]
1. Create a new core bot.
2. Use Rocket menu to hook up LUIS and QnA
3. Run bot
4. Open Webchat
5. DO NOT TYPE ANYTHING YET 

Notice that webchat's first message shows the following message 'Looks like I’m all mixed up. Let’s try asking again, but maybe rephrase your request?

Expected:
Webchat shows the Conversational Core bot's welcome card (visual adaptive card, with indicators on what it can do)

<img width="1299" alt="Webchat_defaultMessage_corebot_2" src="https://user-images.githubusercontent.com/48503434/115158860-280c1480-a045-11eb-8809-560554519034.png">
<img width="399" alt="Webchat_defaultMessage_corebot_1" src="https://user-images.githubusercontent.com/48503434/115158863-293d4180-a045-11eb-9022-16fee0d0e3ff.png">

**Comment by ryanlengel** [comment]
I'm not so sure this is a bug with the template, I created a new conversational bot **but** added my LUIS authoring key directly in bot settings - not the rocket menu. It works that way. 

![image](https://user-images.githubusercontent.com/43043272/115260235-6e19b480-a0e7-11eb-9164-a39511fa9b58.png)

**Comment by ryanlengel** [comment]
I've also tested with the rocket menu, adding an existing LUIS resource, and that worked as well...

@mareekuh did the rocket menu suggest adding QnA? Is see that a QnA trace shows up in your logs but the conversational bot doesn't need that resource. Maybe that's what is mixing it up.

**Comment by Quirinevwm** [comment]
Indeed it looks like QnA ;)

**Comment by ryanlengel** [comment]
I tested adding a QnA key, this worked as expected. I'm on version `v1.4.0-nightly.236490.3735dd3`, @mareekuh it looks like you're on a previous one. Assigning this back to the composer repo!
